lib/player.rb

```diff
@@ -1,11 +1,14 @@
 # frozen_string_literal: true
 
-module Player
-  def position_taken?(board, index)
-    !(board[index].nil? || board[index] == ' ')
+class Player
+  attr_reader :name, :symb
+
+  def initialize(name, symb)
+    @name = name
+    @symb = symb
   end
 
-  def valid_move?(board, index)
-    index.between?(0, 8) && !position_taken?(board, index)
+  def add_symb(symb)
+    @symb = symb
   end
```

Comment:
What does this method do? I do not see it being called anywhere

Reply:
valid_move? is called from main each time a player presses a key to play.
add_symb is not used yet. It will be used to make the player select his/her symbol.